src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityDialectIT.java

```diff
@@ -171,25 +194,35 @@ private static void createRolesAndTenantsTestTable() throws SQLException {
                 + "(19, 'Donkey Inc', 'Wheat', 50, 9223372036854775808, 'RLS_USR_1')");
     }
 
-    private static void createUser(final String userName) throws SQLException {
-        statement.execute("CREATE USER " + userName + " IDENTIFIED BY \"" + userName + "\"");
-        statement.execute("GRANT ALL PRIVILEGES TO " + userName + "");
+    private static void createUser(final List<String> userNames) throws SQLException {
+        for (String userName : userNames) {
+            statement.execute("CREATE USER " + userName + " IDENTIFIED BY \"" + userName + "\"");
+            statement.execute("GRANT ALL PRIVILEGES TO " + userName + "");
+        }
     }
 
-    @Test
-    void testSelectFromExaRlsUsersThrowsException() {
-        assertThrows(SQLException.class, () -> statement.execute("SELECT * FROM EXA_RLS_USERS"));
+    private static Stream<String> withMethodSource() {
```

Comment:
Please rename to `getVirtualSchemaVariants`.

Reply:
fixed